cvat/apps/engine/static/engine/js/annotationUI.js

```diff
@@ -49,24 +49,14 @@
 */
 
 async function initLogger(jobID) {
-    if (!Logger.initializeLogger('CVAT', jobID)) {
+    if (!Logger.initializeLogger(jobID)) {
         const message = 'Could not initialize Logger. Please immediately report the problem to support team';
```

Comment:
const message = 'Logger has been already initialized'

I don't see any others scenarios for that condition.